pkg/recovery/recover.go

```diff
@@ -0,0 +1,370 @@
+// Package recovery provides tooling to help with control plane disaster recovery. Recover() uses a
```

Comment:
A lot of the helper functions in this file are good candidates for unit tests. Can be a todo/follow-up - but we haven't been super good in the past with test coverage on bootkube.